explorer/analyzer/file-formats.md

```diff
@@ -0,0 +1,67 @@
+# Analyzer and file formats
+
+The Analyzer CLI tool makes some assumptions about your project structure and file names for your specification files.
+
+## Filenames
+
+You can include the following in your specification filename, in this order:
+
+```
+$name.$version.$spec.$format
+```
+
+- Name: required
+- Version: optional. Starts with a lowercase `v`. Separate major and minor version numbers with a hyphen `-`. For example, `v1` or `v1-1-0`.
+<!-- TODO: update info on version if it becomes compulsory, e.g. if we decide to ignore version numbers from the info block in the spec -->
+- Spec: optional, but recommended. Stoplight supports `oas` and `schema`.
+- Format: required. Stoplight supports `md`, `json`, `yaml` and `yml`.
+
+### Examples
+
+#### OpenAPI spec (v2 or v3):
+
+```
+# Without versioning
+my-api.oas.json
+my-api.oas.yaml
```

Comment:
I think we'll still allow them to not have version in their filenames. However, that means that spec and nodes within the spec will be un-versioned. Can't remember how un-versioned nodes show up in Explorer in dojo, but should probably mention this consequence.

Reply:
Added a warning up top